BusBoard/src/app.ts

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+import getBuses from "./getBuses";
+import getBusStops from "./getBusStops";
+
+const app = express();
+const port = 3000;
+
+app.get("/", (req, res) => {
+  res.send("Hello World");
+});
+
+app.get("/StopPoint", (req, res) => {
+  res.send("Hello World");
+});
+
+app.get("/departureBoards/:postCode", async (req, res) => {
+  const { postCode } = req.params;
+  const busStops = await getBusStops(postCode);
+
+  const busRequests = busStops.map((busStop) => getBuses(busStop.naptanId));
+
+  const busInfo = await Promise.all(busRequests);
+
+  var obj = {};
```

Comment:
Can you use `let` here, rename `obj` to something more clear, and type the object?